packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts

```diff
@@ -31,6 +32,11 @@ export namespace VscodeCommands {
         label: 'VSCode open link'
     };
 
+    export const DIFF: Command = {
+       id: 'vscode.diff',
+       label: 'VSCode diff'
```

Comment:
@vzhukovskii Do we need it in the command palette?

Reply:
@akosyakov perhaps, no, will provide additional PR to remove this field.

Reply:
Created following PR for this: https://github.com/theia-ide/theia/pull/4450